corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -1280,15 +1280,7 @@ hqDefine("cloudcare/js/formplayer/menus/views", [
         template: _.template($("#breadcrumb-item-template").html() || ""),
         className: "breadcrumb-item",
         attributes: function () {
-            let attributes = {
-                "role": "link",
-                "tabindex": "0",
```

Comment:
are we setting the tabindex somewhere else now?

Reply:
Breadcrumb items use anchor tags now, which are tab indexible by default